vector/v.in.lidar/v.in.lidar.html

```diff
@@ -107,28 +107,28 @@ <h2>EXAMPLE</h2>
 
 This example is analogous to the example used in the GRASS wiki page for
 <a href="https://grasswiki.osgeo.org/wiki/LIDAR#Import_LAS_as_vector_points">importing LAS as vector points</a>.
```

Comment:
This anchor doesn't exist since feb 2012 in https://grasswiki.osgeo.org/w/index.php?title=LIDAR&diff=prev&oldid=14909

Which of https://grasswiki.osgeo.org/wiki/LIDAR#Import_LAS_as_vector_points_in_GRASS_6 or https://grasswiki.osgeo.org/wiki/LIDAR#Import_LAS_in_GRASS_7 is appropriate?

Reply:
I have updated the Wiki page section title to https://grasswiki.osgeo.org/wiki/LIDAR#Import_LAS_in_GRASS_GIS_7_and_later but removed the cross-link from the manual page as the Wiki page is too confusing.